style.css

```diff
@@ -0,0 +1,371 @@
+.headerbackground {
+  display: flex;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  bottom: 20px;
+  background: #fff;
+}
```

Comment:
![white-spacing (copy)](https://user-images.githubusercontent.com/93039359/185142544-e77d55b2-fcd3-4b94-8072-e1019343fdc6.png)

- [ ] You are using fix widths (means widths in px) which is making a white space you can see in your design. Please always make widths in percentage(%) instead of px.
- [ ] Your design is not very reponsive. I know responsiveness in not the requirement yet but your website should expand on your whole screen please fix it by using widths in percentages (%) :blush: